src/unitxt/operators.py

```diff
@@ -858,6 +858,9 @@ def process(self, stream: Stream, stream_name: str = None) -> Generator:
         for instance in stream:
             counter[self.signature(instance)] += 1
 
+        if len(counter) == 0:
```

Comment:
why can we get 0? should we allow that?

Reply:
It can happen if by mistake someone filters too many results.  In most cases, it should not happen, once I added the checks in the standard recipe verify method, but it can happen .    The question is more general - can streams be empty.   But we we allow empty streams, we need operators to behave correctly over them.